tripal/tripal_analysis_blast
# Fixes missing feature viewer & NCBI parsing results

## Description
This PR does two things:
1. It fixes issue #8.   There were problems with the feature viewer not showing up if AJAX loading was turned on for the field.  
2.  It changes the layout to better handle when multiple blast analyses are available by 
    * Adding an accordion.  See screenshot below.
    * Hiding the results table by default.  There is a link under the viewer that let's you see it.
    * Making the features in the browser clickable.  The detailed summary of the hit appears when clicked.
3. It fixes issue #21 by simply using the hit accession as the name rather than the long identifiers.

![image](https://user-images.githubusercontent.com/1719352/50546001-601d8e00-0bd5-11e9-8922-a3bdf5a2d5a8.png)

## How to Test
1. Check out this branch
2.  Run a `drush updatedb`
3.  Go to a page where you have blast results loaded.
4.  Click the 'Reload' button at the top of the page.
5.  Check the blast results link to make sure the viewers are present.
